products/sle/main.pm

```diff
@@ -723,18 +723,20 @@ elsif (get_var("QA_TESTSUITE")) {
     loadtest "qa_automation/execute_test_run";
 }
 elsif (get_var("XFSTESTS")) {
-    #Workaround bsc#1101787
-    if (check_var('ARCH', 'aarch64') && check_var('VERSION', '12-SP4')) {
-        set_var('NO_KDUMP', 1);
+    if (check_var('XFSTESTS', 'installation')) {
+        prepare_target;
+        loadtest "xfstests/install";
+        unless (get_var('NO_KDUMP')) {
+            loadtest "xfstests/enable_kdump";
+        }
+        loadtest "shutdown/shutdown";
     }
-    prepare_target;
-    unless (get_var('NO_KDUMP')) {
-        loadtest "xfstests/enable_kdump";
+    else {
+        prepare_target;
```

Comment:
you can move prepate_target outside the if-else, as both conditions are executing this.

Reply:
thanks, I agree